File: app/src/main/java/br/com/appdesafio/model/persistence/AppDatabase.java
package br.com.appdesafio.model.persistence;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import javax.inject.Singleton;

import br.com.appdesafio.model.entity.UrlEntity;


/**
 * Class that creates the DAO's of the project.
 */
@Singleton
@Database(entities = {UrlEntity.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UrlDAO urlDAO();


}

File: app/src/main/java/com/example/lucasdias/mvvmpoc/data/db/AppDatabase.kt
package com.example.lucasdias.mvvmpoc.data.db

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import com.example.lucasdias.mvvmpoc.domain.entity.PullRequest
import com.example.lucasdias.mvvmpoc.domain.entity.Repository

@Database(version = 1, entities = [Repository::class, PullRequest::class])

    abstract class AppDatabase : RoomDatabase() {
        abstract fun repositoryDao(): RepositoryDao
        abstract fun pullRequestDao(): PullRequestDao
}